test/api/search_test.go

```diff
@@ -43,6 +43,17 @@ func TestSearchApi(t *testing.T) {
 
 	record := lifeline.GetStateRecords()[0]
 	pn := record.Record.ID.Pulse()
+
+	randomJetID := func() string {
+		for {
+			jd := converter.JetIDToString(testutils.GenerateUniqueJetID())
```

Comment:
I don't think its good idea to use `GenerateUniqueJetID` here: you don't need uniqueness here, and just stop other tests to use some generated jetID (that you don't want to use here)